.github/workflows/osgeo4w.yml

```diff
@@ -65,13 +66,15 @@ jobs:
         shell: msys2 {0}
 
       - name: Compile GRASS GIS
-        run: D:\msys64\usr\bin\bash.exe -l (''+(Get-Location)+'\.github\workflows\build_osgeo4w.sh') (Get-Location)
+        shell: msys2 {0}
+        run: .github/workflows/build_osgeo4w.sh
```

Comment:
I'm not sure that this is enough. For the other changes, (msystem and the test simple), I've been trying them on and off since January, and there shouldn't be any problems. Let's see the tests results